src/interlocking_component/route_controller.py

```diff
@@ -131,6 +132,8 @@ def next_tick(self, tick: int):
 
             if was_set:
                 self.routes_to_be_set.remove(interlocking_route)
+        for route in self.routes_to_be_reserved:
+
```

Comment:
I think something is missing here

Reply:
Yes, I don't know why it was not there. Now it is.